pymeasure/instruments/agilent/agilentL4534A.py

```diff
@@ -0,0 +1,333 @@
+        validator=sample_count_function,
+        values=SAMPLE_COUNT_VALUES
+    )
+
+    pre_trigger_samples = Instrument.control(
+        "CONF:ACQ:SPR?",
+        "CONF:ACQ:SPR %d",
+        """Control number of samples that will captured pre-trigger, in multiples of 4 (8-128,000,000)""",
+        validator=sample_count_function,
+        values=PRE_TRIG_VALUES
+    )
+
+    number_of_records = Instrument.control(
+        "CONF:ACQ:REC?",
+        "CONF:ACQ:REQ %d",
+        """Control number of of records that will be captured before arming is disabled (1-1024)""",
+        validator=strict_range,
+        values=RECORD_COUNT_VALUES
+    )
+
+    trigger_holdoff = Instrument.control(
+        "CONF:ACQ:THOL?",
+        "CONF:ACQ:THOL %e",
+        """Control minimum time between triggers, during which trigger inputs are ignored,
+          in decimal seconds (0-10s)""",
+        validator=strict_range,
+        values=TRIGGER_HOLDOFF_VALUES
+    )
+
+    trigger_delay = Instrument.control(
+        "CONF:ACQ:TDEL?",
+        "CONF:ACQ:TDEL %e",
+        """Control delay time after trigger before acquistion starts, in decimal seconds (0-3600s)""",
+        validator=strict_range,
+        values=TRIGGER_DELAY_VALUES
+    )
+
+    maximum_samples = Instrument.measurement(
+        "CONF:ACQ:SCO:MAX?",
+        """Get Maximum number of samples that can be used in a record"""
+    )
+
+    acquisition = Instrument.control(
+        "CONF:ACQ:ATTR?",
+        "CONF:ACQ:ATTR %s",
+        """Control Acquistion settings
+        returns: sample_rate, samples_per_record, pre_trig_samples_per_record, num_records, trigger_holdoff, trigger_delay 
+        """,
+        set_process=_set_acq_config_process,
+        get_process=_get_acq_config_process,
+        validator=_validate_acq_config
+    )
+
+    def arm(self) -> None:
+        """
+        Trigger ARM condition in software
+        """
+        self.write('ARM')
+
+    def init(self) -> None:
+        """
+        Initialize measurement with current configuration.\n
+        This puts the instrument in acquisition state and waits for arm condition.\n
+        If arm is immediate, it will wait for trigger.\n
+        If both arm and trigger are immediate, it will immediately start capturing data.\n
+        Instrument will singal complete once capture is finished
+        """
+        self.write('INIT')
+
+    def auto_zero(self, channels=[1,2,3,4]):
+        """ 
+        Auto-zero the inputs, temporarily loading new offsets for the current config.
+        Offsets will be cleared when instrument is reset or settings are changed.
+        """
+        return int(self.ask('CAL:ZERO:AUTO? (@{})'.format(','.join(channels)), 5).strip())
+    
+    channels = Instrument.ChannelCreator(Channel, (1, 2, 3, 4))
+
+    def __init__(self, adapter, name="Agilent L4534A Digitizer", **kwargs):
+        super().__init__(
+            adapter, name, **kwargs
+        )
```

Comment:
It's best to have the init at the top of a class in order to find it more easily.

Reply:
OK, fixed.